cms/djangoapps/contentstore/views/course.py

```diff
@@ -1317,6 +1331,26 @@ def update_usage_info(store, course, configuration):
         configuration_json['usage'] = usage_information.get(configuration.id, [])
         return configuration_json
 
+    @staticmethod
+    def get_or_create_cohorted_user_partition(course, store, user_id):
```

Comment:
I find the term "cohorted user partition" confusing. It's a user partition with scheme "cohort", and we are referring to it as the partition with "content groups". At the least, how about "cohort_user_partition" instead of "cohorted_user_partition"?